docs/LocalRabbitMqSetup.md

```diff
@@ -0,0 +1,44 @@
+# Setup RabbitMQ Locally
+
+## Install Erlang on Computer
+
+- Download and install [Erlang](https://www.erlang.org/downloads) before you install the
+  [RabbitMQ](https://www.rabbitmq.com/download.html) server
+- For Windows OS, it is recommended to install the 64-bit version
```

Comment:
Which version they install depends on their machine. I think its automatically understood to install the 64-bit version my the machine is 64-bits